The job **`tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/16456615575/job/46515151325) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/46515151325) [(plain)](https://github.com/rust-lang/rust/commit/30033b45bb2fcbc0f267acaf9fab22411422e166/checks/46515151325/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
fmt check
Diff in /checkout/library/core/src/clone.rs:155:
 /// rely on their keys respecting this property for correct behavior.
 ///
 /// This property is automatically satisfied when deriving both `Clone` and [`PartialEq`]
-/// using `#[derive(Clone, PartialEq)]` or when additionally deriving [`Eq`] 
+/// using `#[derive(Clone, PartialEq)]` or when additionally deriving [`Eq`]
 /// using `#[derive(Clone, PartialEq, Eq)]`.
 ///
 /// Violating this property is a logic error. The behavior resulting from a logic error is not
fmt: checked 6213 files
Build completed unsuccessfully in 0:00:45
  local time: Tue Jul 22 22:14:56 UTC 2025
  network time: Tue, 22 Jul 2025 22:14:56 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/library/core/src/clone.rs
+++ b/library/core/src/clone.rs
@@ -160,8 +160,8 @@ mod uninit;
 ///
 /// Violating this property is a logic error. The behavior resulting from a logic error is not
 /// specified, but users of the trait must ensure that such logic errors do *not* result in
-/// undefined behavior. This means that `unsafe` code **must not** rely on the correctness of these
-/// methods.
+/// undefined behavior. This means that `unsafe` code **must not** rely on this property
+/// being satisfied.
 ///
 /// ## Additional implementors
 ///

--- a/library/core/src/clone.rs
+++ b/library/core/src/clone.rs
@@ -176,6 +176,11 @@ mod uninit;
 ///   (even if the referent doesn't),
 ///   while variables captured by mutable reference never implement `Clone`.
 ///
+/// [`HashMap`]: ../../std/collections/struct.HashMap.html
+/// [`HashSet`]: ../../std/collections/struct.HashSet.html
+/// [`BTreeMap`]: ../../std/collections/struct.BTreeMap.html
+/// [`BTreeSet`]: ../../std/collections/struct.BTreeSet.html
+/// [`BinaryHeap`]: ../../std/collections/struct.BinaryHeap.html
 /// [impls]: #implementors
 #[stable(feature = "rust1", since = "1.0.0")]
 #[lang = "clone"]

--- a/library/core/src/clone.rs
+++ b/library/core/src/clone.rs
@@ -155,7 +155,7 @@ mod uninit;
 /// rely on their keys respecting this property for correct behavior.
 ///
 /// This property is automatically satisfied when deriving both `Clone` and [`PartialEq`]
-/// using `#[derive(Clone, PartialEq)]` or when additionally deriving [`Eq`] 
+/// using `#[derive(Clone, PartialEq)]` or when additionally deriving [`Eq`]
 /// using `#[derive(Clone, PartialEq, Eq)]`.
 ///
 /// Violating this property is a logic error. The behavior resulting from a logic error is not

